e2e/cypress/support/ui_commands.js

```diff
@@ -60,14 +60,22 @@ Cypress.Commands.add('uiChangeMessageDisplaySetting', (setting = 'STANDARD') =>
 
 // Type Cmd or Ctrl depending on OS
 Cypress.Commands.add('typeCmdOrCtrl', () => {
+    typeCmdOrCtrlInt('#post_textbox');
+});
+
+Cypress.Commands.add('typeCmdOrCtrlForEdit', () => {
+    typeCmdOrCtrlInt('#edit_textbox');
+});
+
+function typeCmdOrCtrlInt(textboxSelector) {
```

Comment:
I'm a bit lost with "...Int". What doe sit mean? Can we change fo ra meaningful name?

Reply:
@saturninoabril - it is a shortcut for **Int**ernal, but I'm open to different/better naming. The idea is that we don't call the function directly but use the external interface commands.

Reply:
I see, thanks for the explanation.